storybook/src/formik-elements/FieldError.mdx

```diff
@@ -0,0 +1,24 @@
+import { ArgsTable, Story, Canvas, Stories } from "@storybook/addon-docs";
+
+import { FieldError } from "@tiller-ds/formik-elements";
+import { ThemeTokens } from "../utils";
+
+# Field Error
+
+The `FieldError` component allows you to render an error message for a given field.
+
+By defining a name prop on the `FieldError` component, it will automatically render the error message for the field
+with the aforementioned name when there is supposed to be one.
+
+The component is particularly useful if you run into issues with displaying the error message for a field
```

Comment:
This is a bit confusing. FieldError connects to Formik context and shows error. It should not be used as a debugging tool in my opinion.

Reply:
I've noted that because I've seen some teams use it as a tool to help us resolve some bugs related to showing errors on components when errors weren't shown properly.